.github/workflows/get-oidc-token.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  pull_request_target:
+    types: [ opened, edited, synchronize, reopened, ready_for_review ]
+
+jobs:
+  hello_world_job:
+    permissions:
+      id-token: write # This is required for requesting the JWT token
+      contents: read  # This is required for actions/checkout
+    runs-on: ubuntu-latest
+    name: A job to get OIDC token and say hello
+    steps:
+      # This action is used to get the JWT token from the OIDC provider and set is as an output and an environment variable
+      - uses: Sawthis/expose-jwt-action@v1.0.4
+        name: Get JWT token
+        with:
+          aud: 'https://github.com/github'
```

Comment:
Adjust by choosing either 'aud' or 'audience' for consistency. (Line 28)

Reply:
done